crates/ruma-events/src/room/redaction.rs

```diff
@@ -1,5 +1,7 @@
 //! Types for the *m.room.redaction* event.
 
+#![allow(clippy::exhaustive_structs)]
```

Comment:
Since one of the three types in this module is actually non-exhaustive, can just just add this attribute to the other two instead of to the whole module?